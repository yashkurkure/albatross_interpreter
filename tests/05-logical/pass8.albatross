if (!131|226*74*27&161|181*248|223*236&140|141*124*169-240<>171|27|93|176|237*111-121&60|217-55*83&201) {if (!(251|2000000)>251) {if ((53|104-0*134*135|100+64-124+125|2000000)>53|104-0*134*135|100+64-124+125) {if ((3|91&66+251+145&71|242|120-156&110|119|72+121+79|2000000)>3|91&66+251+145&71|242|120-156&110|119|72+121+79) {if (60|141|94-173&30|159*1<(60|141|94-173&30|159*1|2000000)) {printint( 196*167-252*57&3+247|89-162|121*225*193+195|168-155&81-26);}}}}}
if (!200-150+27|126|67&126+159|175|134||1) {if (!27*112|103&20|92|11+1|153&186&112+99+17-174||1) {if ((189|15-81|154*116-210|42*193+218|54-48+72|4+126|2000000)>189|15-81|154*116-210|42*193+218|54-48+72|4+126) {if (!156==156) {if (!16|124+236|254|4|113|20&236|150==16|124+236|254|4|113|20&236|150) {printint( 36|152|155+76-221*96+204+67*191|159&216&249&36|222+157&179|11|107+203|224*43&167|187|216|241);}}}}}
if (!116*81*143*62+185&70|140-189&69*42+49*93|176|41|55&57<>4&78|254+21|178+127*196+180|132+90&208-211+164*34|103) {if (!249&98-41*119*57&154&232&116*241*32|164-176|92-218+12||1) {if ((229&208|167+68*29-236|147+52+88-46|120&47-120*190|207|19|2000000)>229&208|167+68*29-236|147+52+88-46|120&47-120*190|207|19) {if (!61+129&171&105|99==61+129&171&105|99) {if (145+34*60*135&62+202-79|96||1) {printint( 36);}}}}}
if (!137||1) {if (193*193-135+46*198&12|69+107&75*84&8+44&94-70-160|165==193*193-135+46*198&12|69+107&75*84&8+44&94-70-160|165) {if (226|209|20&24&89*65+133|132|183==226|209|20&24&89*65+133|132|183) {if (!147|52+164&202*76|41|104+174&132+192-192+86==147|52+164&202*76|41|104+174&132+192-192+86) {if (!95*61|37*33|156-83|222|183&202-86&120|84&71+130==95*61|37*33|156-83|222|183&202-86&120|84&71+130) {printint( 204+70-52*192|178-229*230&165*210*171|143*95|159|225+119+211|167*152|123|211&33-167|151+226+71);}}}}}
if (!160&86&21|162|36*201|213*64&210<(160&86&21|162|36*201|213*64&210|2000000)) {if (!47+86|48+150&156+170-184&201+217|224&20*165+27==47+86|48+150&156+170-184&201+217|224&20*165+27) {if ((183*188&70|4&50|54&202|198+137-91|85|184|2000000)>183*188&70|4&50|54&202|198+137-91|85|184) {if ((171|152|244*137&159-236|55*182|2000000)>171|152|244*137&159-236|55*182) {if (!242+143<>121|120|222+191|36*72|204+54+149|8+76-143+121) {printint( 11&211&211|3|10+202|167&213|102*77&219);}}}}}
if (243+116|110&21*39&7|96|56-31|213|223*238*85+232-226==243+116|110&21*39&7|96|56-31|213|223*238*85+232-226) {if (155&183|176*158|204*164+213|145-6==155&183|176*158|204*164+213|145-6) {if (187*5|71|36*136+156*214*68|224|35+210&172&231==187*5|71|36*136+156*214*68|224|35+210&172&231) {if (!88-20|146|92|237|195*160-150|183|59+216&93-236<(88-20|146|92|237|195*160-150|183|59+216&93-236|2000000)) {if (226*204*52*172+45*171*9|56|16*84|251|91|74&187<(226*204*52*172+45*171*9|56|16*84|251|91|74&187|2000000)) {printint( 227);}}}}}
if (145*253|201+245-111|189-121*165|35|179-36|105*110+23|13==145*253|201+245-111|189-121*165|35|179-36|105*110+23|13) {if (234*31-20|96&96-209-34|45|39|138|119|133==234*31-20|96&96-209-34|45|39|138|119|133) {if (55|37*123-45-67-95+76|12+26-152&20|238&173==55|37*123-45-67-95+76|12+26-152&20|238&173) {if (142|47-128-211|251-225-243*182&225-177*6-102&53-14*175|225||1) {if (!221*158|42-53+120*3|223*40+169+235|39+10|136-243*117==221*158|42-53+120*3|223*40+169+235|39+10|136-243*117) {printint( 253);}}}}}
if (!156-169+134-92|147-184*176*160|66&143|188*22+201+144+89+153==156-169+134-92|147-184*176*160|66&143|188*22+201+144+89+153) {if (17|160|238|95+37*103-161-6+143==17|160|238|95+37*103-161-6+143) {if (!92|49|130|64-212+243*26+127*123+205*19|138*48&54|159|230==92|49|130|64-212+243*26+127*123+205*19|138*48&54|159|230) {if (!235&115|39+171&35+240|213&255+132*3+233-16+20|182&20&80<>148|222|230*41|60&247-102*245&101) {if (!247==247) {printint( 191-60*158-249&105+234|137-73&149*150*200*192|148&54|169&242*17+10*87|177);}}}}}
if (216|157&210+191*78|64|131&147*85+193|147|202|57|23&224<>43&191*61|83-67+200|232|97&163|72-149|181*200+89|7|126) {if (!63&212&239|142-144|166+192+251|90|135|221|142||1) {if (18*48|139|57+107&93+6-198-22*162|249|45&74||1) {if (!48-153+173&232*122|179-253|69|108|212&84+216-252|129|133||1) {if (198|48*13|253|208|158+103-146+2|29&192*53|44-75|179|104==198|48*13|253|208|158+103-146+2|29&192*53|44-75|179|104) {printint( 16|75-193+141|31|93|3-30|58|191-37|43*163|84&192|247*156*140|114-93&231&46*99+59+32-220|212+62-77);}}}}}
if (254|37|90|135|183|148|62|140-179&90==254|37|90|135|183|148|62|140-179&90) {if (104*142|15*140*68|226||1) {if (!(226+41-47-90+128*16|191+213+99|2000000)>226+41-47-90+128*16|191+213+99) {if (31|178-74-140*47|13|134==31|178-74-140*47|13|134) {if (203|140+255&155|127&240-132|1&188|108*3-68==203|140+255&155|127&240-132|1&188|108*3-68) {printint( 167|169-71|33-68+99&249|237*213-250&221|60&197|135|25|219|149|95|206|204-245&124);}}}}}
exit(0);
