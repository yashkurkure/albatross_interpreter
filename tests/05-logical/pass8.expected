36-1223227253-1255
